### ISSUE TITLE
【Question】About dmd train

### ISSUE THREAD

**Issue by ApolloRay** [Zeqiang-Lai/OpenDMD#6]
During the DMD training stage, the loss g and loss d did not drop significantly. Can you provide a loss decline trend during the training process?
![截屏2024-05-15 11 59 44](https://github.com/Zeqiang-Lai/OpenDMD/assets/36093263/5543822e-9dd0-43aa-bee8-7eb7c02695f0)
![截屏2024-05-15 12 00 06](https://github.com/Zeqiang-Lai/OpenDMD/assets/36093263/e30617a3-db4a-4cf9-8bf0-78006cb15b6e)

**Comment by ApolloRay** [comment]
I use about 25W data (generated by sd1.5) to train the dmd model.

**Comment by Zeqiang-Lai** [comment]
Unfortunately, the printing log is not saved...
This is normal as loss d is mse diffusion loss, it always fluctuates. You could track the training process by looking at the saved val image to see if the quality is improved.

**Comment by ApolloRay** [comment]
> Unfortunately, the printing log is not saved... This is normal as loss d is mse diffusion loss, it always fluctuates. You could track the training process by looking at the saved val image to see if the quality is improved.

Image quality have just improved little. I'm not sure whether the image quality generated by teacher model influence it. I train 70000 steps, generated images are blurred.

**Comment by ApolloRay** [comment]
Hi, the dreamshaper_8_6k means the gstep=6000 ?How about the batch size and the weight of kl or reg_loss ?

**Comment by ApolloRay** [comment]
In your train.sh， I found you have sent the reg_loss_weight=0， so the final checkpoints is in this setting ？

**Comment by ApolloRay** [comment]
I have train the dreamshaper 8 in 170000 steps, but the g_loss and d_loss, always keep in the range(1.34-1.42)，(0.12-0.14)
<img width="683" alt="截屏2024-06-04 17 09 18" src="https://github.com/Zeqiang-Lai/OpenDMD/assets/36093263/fcf5fa02-3ada-409f-a582-e1125a468a90">

**Comment by ApolloRay** [comment]
Hope to add your wechat to contact about this model.

**Comment by Zeqiang-Lai** [comment]
Could you provide more visual results you obtained ?

BTW: the official training code of DMD2 is released.

Here are some results obtained from a model I trained before, though the base model is not dreamshaper8.

<img width="805" alt="image" src="https://github.com/Zeqiang-Lai/OpenDMD/assets/26198430/b46bd3b0-f2c8-4a97-9900-7d2484a570db">

**Comment by ApolloRay** [comment]
> Could you provide more visual results you obtained ?
> 
> BTW: the official training code of DMD2 is released.
> 
> Here are some results obtained from a model I trained before, though the base model is not dreamshaper8.
> 
> <img alt="image" width="805" src="https://private-user-images.githubusercontent.com/26198430/337548770-b46bd3b0-f2c8-4a97-9900-7d2484a570db.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc3NDE3OTIsIm5iZiI6MTcxNzc0MTQ5MiwicGF0aCI6Ii8yNjE5ODQzMC8zMzc1NDg3NzAtYjQ2YmQzYjAtZjJjOC00YTk3LTk5MDAtN2QyNDg0YTU3MGRiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA3VDA2MjQ1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlkMjcxYTE1MDA2YzlmYTk2MzUzYjI4MGIxZDczOWVmOTNhNGJlOWI4YmRhZjNlM2EzMmQzNjkzYzJhYjBmZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nIpuB2es39n1EveDAThty3_I7NSwwcRKqOFL8j3XAuU">

Really cool. I will sort out all the relevant information and inference results of my current training, and I will update it under this issue. I will also take a look at the DMD2 repo.
